picard/ui/itemviews.py

```diff
@@ -1160,47 +1168,46 @@ def update(self, update_track=True, update_selection=True):
             self.setText(i, file.column(column[1]))
             self.setForeground(i, color)
             self.setBackground(i, bgcolor)
-        if file.errors:
-            self.setToolTip(MainPanel.TITLE_COLUMN, _("Processing error(s): See the Errors tab in the File Info dialog"))
         if update_selection and self.isSelected():
             TreeItem.window.update_selection(new_selection=False)
         parent = self.parent()
         if isinstance(parent, TrackItem) and update_track:
             parent.update(update_files=False, update_selection=update_selection)
 
     @staticmethod
-    def decide_file_icon(file):
+    def decide_file_icon_info(file):
+        tooltip = ""
         if file.state == File.ERROR:
             if file.error_type == FileErrorType.NOTFOUND:
-                return FileItem.icon_error_not_found
+                icon = FileItem.icon_error_not_found
+                tooltip = _("File not found")
             elif file.error_type == FileErrorType.NOACCESS:
-                return FileItem.icon_error_no_access
+                icon = FileItem.icon_error_no_access
+                tooltip = _("File permission error")
             else:
-                return FileItem.icon_error
+                icon = FileItem.icon_error
+                tooltip = _("Processing error(s): See the Errors tab in the File Info dialog")
```

Comment:
I would be tempted to set the text to "Processing error: See the Errors tab in the File Info dialog", only because I don't know how well "error(s)" translates to other languages.

Reply:
I don't think that has much impact on translation. If that's uncommon in the target language or the target language has no plural forms then they would just not have any special form for this. That's part of the translation.

Reply:
Okay, that makes sense.  Thanks for clarifying.